src/Microsoft.Cci.Extensions/Differs/ElementDifferenceFactory.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Contracts;
+using System.Linq;
+using Microsoft.Cci.Mappings;
+
+#if COREFX
```

Comment:
MEF is an interesting dependency. We may be able to switch to System.Composition all the time for our usages in this library.